contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,166 @@
+# See readme.md for instructions on running this code.
+import requests
+import json
+import os
+
+index_error_happened = False
+
+
+class cd:
+    """Context manager for changing the current working directory"""
+    def __init__(self, newPath):
+        self.newPath = os.path.expanduser(newPath)
+
+    def __enter__(self):
+        self.savedPath = os.getcwd()
+        os.chdir(self.newPath)
+
+    def __exit__(self, etype, value, traceback):
+        os.chdir(self.savedPath)
+
+
+class GitHubHandler(object):
+    '''
+    This plugin allows you to comment on a GitHub issue, under a certain repository.
+    It looks for messages starting with '@comment' or '@gcomment'.
+    '''
+
+    def usage(self):
+        return '''
+            This bot will allow users to comment on a GitHub issue.
+            Users should preface messages with '@comment' or '@gcomment'.
+            You will need to have a GitHub account.
```

Comment:
GitHub tokens have different privileges, you might want to specify which privilege is needed for running this bot.